### DISCUSSION TITLE
::marker styling for list items

### DISCUSSION THREAD

**Discussion by schniper** [tailwindlabs/tailwindcss#6574] [Help]
In 3.0, prose ::before list item styling has been replaced by ::marker. But marker is a lot more limited - we can only change the color and font size, which causes missalignments with the list item text.
I have implemented a design with large bullets and larger padding in 2.2, which with ::marker is just impossible to get right.
I have managed to load the default prose styling and delete the marker sections than reimplement ::before ones, but this is quite cumbersome.
What do you guys think? The smaller code footprint for ::marker seems to be outweighed by the flexibility of ::before. 

An idea would be to have a class that would tell the list to skip the ::marker markup or one to use ::before instead. But this would need to happen at base prose styling level for ease of use.

You can’t use ::before over ::marker, as the latter takes precedence, apparently.

**Comment by schniper** [comment]
![077AFDAD-669C-4E8D-A6DA-926667CB770C](https://user-images.githubusercontent.com/533628/146466339-5e0a1a19-db99-42bc-b969-904dd8c20952.jpeg)

this is an example of what I can’t get working with markers. Just playing with the marker font size and line height messes up the alignment with the text.

can use this page as a code example: https://christiantour.ro/seniorvoyage/turism/barcelona-si-catalonia-1110

**Comment by adamwathan** [comment]
Hey! You can always override the default styles for the typography plugin in your config file to use `::before` if you prefer 👍🏻

**Comment by schniper** [comment]
I was under the impression that if I set a key that already exists, the clauses inside the overwritten one get extended by the new ones - didn’t bother ti check the source code. I’ll try overwriting the default clauses. Thanks!

—
Costin Bereveanu
________________________________
